www/src/pages/guidelines/design-tokens.js

```diff
@@ -0,0 +1,175 @@
+import React from "react"
+import MdWarning from "react-icons/lib/md/warning"
+
+import {
+  Container,
+  Section,
+  Columns,
+  ContentColumn,
+  CopyColumn,
+} from "../../components/guidelines/containers"
+import {
+  Intro,
+  PageHeading,
+  SectionHeading,
+} from "../../components/guidelines/typography"
+import Layout from "../../components/guidelines/layout"
+import Badge from "../../components/guidelines/badge"
+
+import theme from "../../utils/guidelines/theme"
+import { Box } from "../../components/guidelines/system"
+
+const DesignTokens = ({ location }) => (
+  <Layout pathname={location.pathname}>
+    <Container>
+      <div css={{ position: `relative`, zIndex: 1 }}>
+        <PageHeading>Design Tokens</PageHeading>
+        <Intro>
+          This page collects all design tokens currently available for
+          gatsbyjs.org which are not covered in "Color" or "Typography"
+        </Intro>
+        <Badge my={3}>
+          Work in Progress{` `}
+          <MdWarning style={{ fontSize: 16, marginLeft: `0.25rem` }} />
+        </Badge>
+      </div>
+    </Container>
+
+    <Section>
+      <SectionHeading>Space</SectionHeading>
+      <Columns>
+        <CopyColumn>
+          <p>
+            Spacing tokens are intended for use with <code>margin</code>,{` `}
+            <code>padding</code>, and other layout-related CSS properties.
+          </p>
+          <p>
+            The primary use for these tokens should be building individual
+            components; however, as we flesh out this part of our design tokens,
+            they are also expected to be used to define global (layout) spacing
+            properties.
+          </p>
+        </CopyColumn>
+        <ContentColumn>
+          <table>
+            <thead>
+              <tr>
+                <th>Token</th>
+                <th>rem</th>
+                <th>px</th>
+                <th>Example</th>
+              </tr>
+            </thead>
+            <tbody>
+              {theme.space.map((space, index) => (
+                <tr key={`tokens-space-${index}`}>
+                  <td>
+                    <code>space[{index}]</code>
+                  </td>
+                  <td>{space.substring(0, space.length - 3)}</td>
+                  <td>{parseFloat(space) * 16}</td>
+                  <td>
+                    {` `}
+                    <Box
```

Comment:
We should label these examples somehow. Visually hidden text describing them? That will hopefully keep this error from being reported https://dequeuniversity.com/rules/axe/3.2/th-has-data-cells?application=AxeChrome

Reply:
> Visually hidden text describing them?

I added that in a772618. From what I understand this should make this pass the Axe test. However I'm wondering if we need something similar for the other examples, too — for people using screenreaders, would adding a description, e.g. "This table cell contains text rendered with `lineHeight.dense`, along hiding the copy demonstrating the `line-height` with `aria-hidden`, make sense?